types/index.d.ts

```diff
@@ -1,2 +1,6 @@
-import {Vue} from "./vue.d";
+import {Vue as _Vue} from "./vue.d";
```

Comment:
Yes, it should.

@kaorun343 would you please have a look on this? I don't want to solve two different issues in one pull request.